cuenca/resources/wallet_transactions.py

```diff
@@ -0,0 +1,35 @@
+from typing import Optional, cast
+
+from cuenca_validations.types import (
+    WalletTransactionRequest,
+    WalletTransactionType,
+)
+
+from cuenca.resources.base import Creatable, Transaction
+from cuenca.resources.resources import retrieve_uri
+
+from .base import Wallet
+
+
+class WalletTransaction(Transaction, Creatable):
+    _resource = 'wallet_transactions'
+    type: WalletTransactionType
+    wallet_uri: str
+
+    @property
+    def wallet(self) -> Optional['Wallet']:
+        return cast('Wallet', retrieve_uri(self.wallet_uri))
```

Comment:
does this work?

`Wallet` isn't here: https://github.com/cuenca-mx/cuenca-python/blob/0a3a0409a24838227059b04ccaf5418519438b69/cuenca/resources/__init__.py#L45-L66

Reply:
`Wallet` es la clase base,  `Saving` es el resource